test/quicksiler-config-client.test.js

```diff
@@ -29,8 +29,8 @@ describe('QuicksilverConfigClient', () => {
       (() => new QuicksilverConfigClient()).should.throw(TypeError);
     });
 
-    // Check API base URL.
-    it('base URL option should be set', () => {
+    // Check QS_CONFIG_PATH has a value.
+    it('QS_CONFIG_PATH in .env should be set', () => {
```

Comment:
In terms of the test, does `process.env.QS_CONFIG_PATH.should.be.not.empty()` still make sense regardless of if the value comes from `.env` or "a normal environment variable"? Should this test check for the existance of at least one?